tests/performance/scripts/dcluster_run.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+################################################################################
+# [Fragment] Marathon in Development (Local) Cluster
+# ------------------------------------------------------------------------------
+# This script runs the scale tests on the cluster deployed in the previous steps
+################################################################################
+
+# Validate environment
+if [ -z "$TESTS_DIR" ]; then
+  echo "ERROR: Required 'TESTS_DIR' environment variable"
+  exit 253
+fi
+if [ -z "$CLUSTER_CONFIG" ]; then
+  echo "ERROR: Required 'CLUSTER_CONFIG' environment variable"
+  exit 253
+fi
+if [ -z "$MARATHON_VERSION" ]; then
+  echo "ERROR: Required 'MARATHON_VERSION' environment variable"
+  exit 253
+fi
+# if [ -z "$DATADOG_API_KEY" ]; then
```

Comment:
remove this comment block?

Reply:
Nope, remove comment 😄 I commented this out for debugging the script locally...